helm/cluster-operator-chart/templates/configmap.yaml

```diff
@@ -11,6 +11,8 @@ data:
           server: true
       listen:
         address: 'http://0.0.0.0:8000'
+    controlPlane:
+      subnets: '{{ if .Values.Installation.V1.GiantSwarm }}{{ if .Values.Installation.V1.GiantSwarm.ClusterOperator }}{{ range $index, $element := .Values.Installation.V1.GiantSwarm.ClusterOperator.ControlPlaneSubnets }}{{if $index}} {{end}}{{$element}}{{end}}{{end}}{{end}}'
```

Comment:
I needed all of my 3 braincells to follow this bad boy. I don't have an idea how to improve this but it is not easy to read.

Reply:
Me neither can understand, can we formatting it?

Reply:
I understand your concern. I'll drop those ifs later after this https://github.com/giantswarm/installations/pull/777 will be deployed. For now I just want to be sure that merge will not break not-updated envs